java/src/main/java/com/powsybl/dataframe/network/extensions/CgmesMetadataModelDataframeProvider.java

```diff
@@ -0,0 +1,70 @@
+package com.powsybl.dataframe.network.extensions;
+
+import com.google.auto.service.AutoService;
+
+import com.powsybl.cgmes.model.CgmesMetadataModel;
+import com.powsybl.commons.PowsyblException;
+import com.powsybl.dataframe.network.ExtensionInformation;
+import com.powsybl.dataframe.network.NetworkDataframeMapper;
+import com.powsybl.dataframe.network.NetworkDataframeMapperBuilder;
+import com.powsybl.dataframe.network.adders.NetworkElementAdder;
+import com.powsybl.iidm.network.Network;
+import com.powsybl.cgmes.extensions.CgmesMetadataModels;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Stream;
+
+/**
+ * @author Naledi El Cheikh <naledi.elcheikh@rte-france.com>
+ */
+
+@AutoService(NetworkExtensionDataframeProvider.class)
+public class CgmesMetadataModelDataframeProvider extends AbstractSingleDataframeNetworkExtension {
+    @Override
+    public String getExtensionName() {
+        return CgmesMetadataModels.NAME;
+    }
+
+    @Override
+    public ExtensionInformation getExtensionInformation() {
+        return new ExtensionInformation(CgmesMetadataModels.NAME, "Provides information about CGMES metadata models",
+                "index : id (str), cgmes_subset (str), id (str), description (str), " +
+                        "version (int), modeling_authority_set (str), profiles (str), dependent_on (str), supersedes (str) ");
+    }
+
+    private Stream<CgmesMetadataModel> itemsStream(Network network) {
+        if (network.getExtension(CgmesMetadataModels.class) == null) {
+            throw new PowsyblException("No CGMES metadata model found");
```

Comment:
To avoid throwing error if there is no extension, it would be better to just return no element :
```suggestion
            return Stream.empty()
```
This way in the python we only get an empty dataframe